pyproject.toml

```diff
@@ -16,6 +16,12 @@ pytest-cov = "^3.0.0"
 pylint = "^2.13.4"
 black = "^22.3.0"
 pydocstyle = "^6.1.1"
+mkdocs = {version = "^1.3.0"}
+mkdocs-material = {version = "^8.2.9"}
+mkdocstrings = {extras = ["python"], version = "^0.18.1"}
+mkdocs-gen-files = {version = "^0.3.4"}
+mkdocs-typer = {version = "^0.0.2"}
```

Comment:
Is there any reason to use the more verbose version specification here? Aside from `mkdocstrings`, I think everything should be simplified if possible.